oracle/pkg/updater/updater.go

```diff
@@ -327,16 +334,17 @@ func (u *Updater) handleOpenedCommitment(
 	commitmentTxnHashes := strings.Split(update.TxnHash, ",")
 	// Ensure Bundle is atomic and present in the block
 	for i := 0; i < len(commitmentTxnHashes); i++ {
-		posInBlock, found := txns[commitmentTxnHashes[i]]
-		if !found || posInBlock != txns[commitmentTxnHashes[0]]+i {
+		txnDetails, found := txns[commitmentTxnHashes[i]]
+		if !found || txnDetails.PosInBlock != (txns[commitmentTxnHashes[0]].PosInBlock)+i || !txnDetails.Succeeded {
```

Comment:
Could be easier to read this line if using a descriptive var like `inExpectedPosition`

Reply:
the expected pos in this case would actually be: `txns[commitmentTxnHashes[0]].PosInBlock+i`